assets/RNAseq_report.Rmd

```diff
@@ -1209,14 +1203,15 @@ for (i in 1:length(contrast_names)){
 ```
 
 ```{r load_config, bootstrap.show.message=FALSE, echo=FALSE, message=FALSE, warning=FALSE}
-log_FC_text <- as.character(params$log_FC_threshold)
+logFC_text <- as.character(params$logFC_threshold)
+pval_text <- as.character(params$pval_threshold)
 ```
 
 ## List of differentially expressed genes
 
 The analysis of the differential gene expression was performed using `DESeq2`.
-Genes were considered differentially expressed (DE) when the adjusted p-value was lower than 0.05 (padj < 0.05).
-Genes were further considered differentially expressed (DE) if the log2 Fold Change in expression between the two considered groups was above the threshold of `r log_FC_text` in absolute terms.
+Genes were considered differentially expressed (DE) when the adjusted p-value was lower than `r pval_text` (padj < r `pval_text`).
```

Comment:
Are you going to exchange that `r pval_text` string  by the user input pval_threshold?